charts/xp-management/templates/_helpers.tpl

```diff
@@ -144,7 +156,7 @@ SegmenterConfig:
     MinS2CellLevel: 10
     MaxS2CellLevel: 14
 MLPConfig:
```

Comment:
Shall we also set this as `MlpConfig` ? If not, the `| mergeOverwrite $defaultConfig` below will cause problems, right?

Reply:
I think xp is expecting it as MLPConfig only. (https://github.com/caraml-dev/xp/blob/main/management-service/config/config.go#L21).

Reply:
But yeah, I can change it to `MLPConfig` everywhere to avoid manual problems. A user can add it in values to override the generated value.

Reply:
updated it to `MlpConfig`.